packages/suite-desktop/src-electron/electron.ts

```diff
@@ -214,10 +214,10 @@ const init = async () => {
                 callback(url);
             });
         }
-
-        registerShortcuts(mainWindow);
     }
 
+    registerShortcuts(mainWindow);
```

Comment:
`registerShortcuts` should be used only in dev, no?

Reply:
It used to be only for prod but I think it doesn't really matter if the shortcuts are available in dev too.